app/src/main/java/com/orange/ods/app/ui/about/AboutNavGraph.kt

```diff
@@ -15,13 +15,16 @@ import androidx.navigation.NavType
 import androidx.navigation.compose.composable
 import androidx.navigation.navArgument
 import com.orange.ods.app.ui.LocalMainTabsManager
+import com.orange.ods.app.ui.LocalMainTopAppBarManager
 import com.orange.ods.app.ui.MainDestinations
+import com.orange.ods.app.ui.MainTopAppBarState
 
 fun NavGraphBuilder.addAboutGraph() {
     composable(
         "${MainDestinations.AboutItemDetailRoute}/{${MainDestinations.AboutItemIdKey}}",
         arguments = listOf(navArgument(MainDestinations.AboutItemIdKey) { type = NavType.LongType })
     ) { backStackEntry ->
+        LocalMainTopAppBarManager.current.updateTopAppBar(MainTopAppBarState.DefaultConfiguration)
         LocalMainTabsManager.current.clearTopAppBarTabs()
```

Comment:
We can maybe create a `reset` function to apply default configuration and clear the tabs cause it is done many times. What do you think?

Reply:
I'm not really fan of ` ResetTopAppBar()` composable. I suggest another thing I tried on my side (I stash my work, it works). I added the tabs management in the top app bar cause tabs are for me a part of this bar. I'll show you when you'll be back.